paper-scroll-header-panel.html

```diff
@@ -0,0 +1,450 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<!--
+`paper-scroll-header-panel` contains a header section and a content section.  The
+header is initially on the top part of the view but it scrolls away with the
+rest of the scrollable content.  Upon scrolling slightly up at any point, the
+header scrolls back into view.  This saves screen space and allows users to
+access important controls by easily moving them back to the view.
+
+__Important:__ The `paper-scroll-header-panel` will not display if its parent does not have a height.
+
+Using [layout attributes](http://www.polymer-project.org/docs/polymer/layout-attrs.html), you can easily make the `paper-scroll-header-panel` fill the screen
+
+    <body fullbleed layout vertical>
+      <paper-scroll-header-panel flex>
+        <paper-toolbar>
+          <div>Hello World!</div>
+        </paper-toolbar>
+      </paper-scroll-header-panel>
+    </body>
+
+or, if you would prefer to do it in CSS, just give `html`, `body`, and `paper-scroll-header-panel` a height of 100%:
+
+    html, body {
+      height: 100%;
+      margin: 0;
+    }
+    paper-scroll-header-panel {
+      height: 100%;
+    }
+
+`paper-scroll-header-panel` works well with `paper-toolbar` but can use any element
+that represents a header by adding a `core-header` class to it.
+
+    <paper-scroll-header-panel>
+      <paper-toolbar>Header</paper-toolbar>
+      <div>Content goes here...</div>
+    </paper-scroll-header-panel>
+
+Styling scroll-header-panel:
+
+  To change background for toolbar when it is at its full size:
+
+  paper-scroll-header-panel {
+    --paper-scroll-header-panel-full-header: {
+      background-color: red;
+    };
+  }
+
+  To change the background for toolbar when it is condensed:
+
+  paper-scroll-header-panel {
+    --paper-scroll-header-panel-condensed-header: {
+      background-color: #f4b400;
+    };
+  }
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-resizable-behavior/iron-resizable-behavior.html">
+
```

Comment:
Docs are not showing up. <!-- documentation -> must be right above dom-module. Move <link> elements above the docs.